tsconfig.json

```diff
@@ -0,0 +1,3 @@
+{
+  "compilerOptions": { "types": ["./deno.d.ts"] }
```

Comment:
remove this file

Reply:
@ry Oh, I'm sorry. I'm using this for types in VSCode